src/components/NcButton/NcButton.vue

```diff
@@ -352,6 +440,15 @@ export default {
 				on: {
 					...this.$listeners,
 					click: ($event) => {
+						// Update pressed prop on click if it is set
+						if (typeof this.$props.pressed === 'boolean') {
```

Comment:
```suggestion
						if (typeof this.pressed === 'boolean') {
```

Reply:
It seems to me, we always use shorthand props access

Reply:
ok